### ISSUE TITLE
Suggestion: trace dependency graph

### ISSUE THREAD

**Issue by OliverJAsh** [plantain-00/type-coverage#1]
#### Version(if relevant): 1.0.8

When a `tsconfig.json` specifies `files`, TypeScript will trace the dependency graph of those files, and include all files in the build.

We utilise this so we only have to specify the application entry point.

It seems that type-coverage doesn't follow this pattern, however. It only reports type coverage of the entry files, and not the dependencies.

**Comment by plantain-00** [comment]
v1.0.9 should do that now.

**Comment by OliverJAsh** [comment]
Thanks!

After upgrading, now I'm getting this error:

```
❯ type-coverage
module.js:545
    throw err;
    ^

Error: Cannot find module 'tslib'
    at Function.Module._resolveFilename (module.js:543:15)
    at Function.Module._load (module.js:470:25)
    at Module.require (module.js:593:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/OliverJAsh/Development/unsplash-web/node_modules/type-coverage/dist/index.js:3:17)
    at Module._compile (module.js:649:30)
    at Object.Module._extensions..js (module.js:660:10)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:501:12)
    at Function.Module._load (module.js:493:3)
```

**Comment by plantain-00** [comment]
just released v1.0.10, it should be OK now
